skrifa/src/metrics.rs

```diff
@@ -545,6 +565,29 @@ mod tests {
         assert_eq!(expected, &result[..]);
     }
 
```

Comment:
We changed all that scaling and the only test we need is that Size::unscaled() and Size::new(upem) are the same?

Shouldn't there be a unit test that we fail without your modifications to prove the scaling works?

Reply:
At the very least a test of some scale factor applications?

Reply:
Oops, I forgot to note that I’m going to change the glyph extraction script to grab advances too so that the tests are data driven. I’ll do a follow up for that. 

I’ll add an additional unit test here that specifically tests the scaling code with the values reported in the issue.